### PR TITLE
Do not profile setup in benchmarks

### PR DESCRIPTION
**Pull request by Sbozzolo**
Split set up and runtime phases in benchmarks so that we can get a more accurate reading for the runtime costs

### CODE REVIEW

File: experiments/benchmarks/bucket.jl [hunk 1]
```diff
@@ -141,9 +141,7 @@ function setup_prob(t0, tf, Δt; nelements = (100, 10))
     return prob, cb
 end
 
-function setup_and_solve_problem(; greet = false)
-    # We profile the setup phase as well here. This is not intended, but it is the easiest
-    # to set up for both CPU/GPU at the same time
+function setup_one(; greet = false)
```
Comment (kmdeck) [inline experiments/benchmarks/bucket.jl:144]: why this function name?

**Review by kmdeck — APPROVED** [review]
looks good. thank you!

File: experiments/benchmarks/bucket.jl (hunk 1 above)
Comment (Sbozzolo) [reply experiments/benchmarks/bucket.jl:144]: `one`, as one instance of a problem that is fully ready to be solved

Comment (kmdeck) [reply experiments/benchmarks/bucket.jl:144]: perhaps "setup_simulation" is clearer, or "setup_instance"? as you prefer.

**Pull request merged**